modules/web/server/src/main/scala/observe/web/server/http4s/ObserveCommandRoutes.scala

```diff
@@ -100,101 +89,86 @@ class ObserveCommandRoutes[F[_]: Async: Compression](
     //     ) / BooleanVar(bp) as user =>
     //   se.setSkipMark(inputQueue, obsId, user, obs, stepId, bp) *>
     //     Ok(s"Set skip mark in step $stepId of sequence $obsId")
-    //
-    // case POST -> Root / ObsIdVar(obsId) / StepId(stepId) / "stop" / ObserverVar(
-    //       obs
-    //     ) as user =>
-    //   se.stopObserve(inputQueue, obsId, obs, user, graceful = false) *>
-    //     Ok(s"Stop requested for $obsId on step $stepId")
-    //
-    // case POST -> Root / ObsId(obsId) / StepId(stepId) / "stopGracefully" / ObserverVar(
-    //       obs
-    //     ) as user =>
-    //   se.stopObserve(inputQueue, obsId, obs, user, graceful = true) *>
-    //     Ok(s"Stop gracefully requested for $obsId on step $stepId")
-    //
-    // case POST -> Root / ObsId(obsId) / StepId(stepId) / "abort" / ObserverVar(obs) as user =>
-    //   se.abortObserve(inputQueue, obsId, obs, user) *>
-    //     Ok(s"Abort requested for $obsId on step $stepId")
-    //
-    // case POST -> Root / ObsId(obsId) / StepId(stepId) / "pauseObs" / ObserverVar(obs) as user =>
-    //   se.pauseObserve(inputQueue, obsId, obs, user, graceful = false) *>
-    //     Ok(s"Pause observation requested for $obsId on step $stepId")
-    //
-    // case POST -> Root / ObsId(obsId) / StepId(stepId) / "pauseObsGracefully" / ObserverVar(
-    //       obs
-    //     ) as user =>
-    //   se.pauseObserve(inputQueue, obsId, obs, user, graceful = true) *>
-    //     Ok(s"Pause observation gracefully requested for $obsId on step $stepId")
-    //
-    // case POST -> Root / ObsIdVar(obsId) / StepId(stepId) / "resumeObs" / ObserverVar(obs) as user =>
-    //   se.resumeObserve(inputQueue, obsId, obs, user) *>
-    //     Ok(s"Resume observation requested for $obsId on step $stepId")
-    //
+
+    case req @ POST -> Root / ObsIdVar(obsId) / StepIdVar(stepId) / "stop" /
```

Comment:
I think we should include `clientId` in all endpoints, even if it just for logging

Reply:
Good idea